sale_report_hide_price/models/sale.py

```diff
@@ -0,0 +1,19 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    show_price = fields.Boolean(
+        "Show Price",
+        help="If checked, you can see the price in "
+        "report of"
+        "Sales Order / Quotation.",
+    )
+    show_discount = fields.Boolean(
+        "Show Discount",
+        help="If checked, you can see the discount"
+        " in report of Sales Order/Quotation.",
+    )
```

Comment:
@ForamSerpentCS We should have this feature linked with Security groups.
These fields should be seen by ACCOUNT Manager (or top account group).

Reply:
Yes we may, If we can have security groups for it, then we can manage this feature directly applying field security in template.